src/Files.App/Actions/Content/Tags/OpenAllTaggedActions.cs

```diff
@@ -33,13 +33,18 @@ public OpenAllTaggedActions()
 
 		public async Task ExecuteAsync()
 		{
-			var files = _tagsContext.TaggedItems.Where(item => !item.isFolder);
-			var folders = _tagsContext.TaggedItems.Where(item => item.isFolder);
+			var files = _tagsContext.TaggedItems.Where(item => !item.isFolder).Select(f => f.path);
+			var folders = _tagsContext.TaggedItems.Where(item => item.isFolder).Select(f => f.path);
 
-			await Task.WhenAll(files.Select(file 
-				=> NavigationHelpers.OpenPath(file.path, _pageContext.ShellPage!)));
-
-			folders.ForEach(async folder => await NavigationHelpers.OpenPathInNewTab(folder.path, false));
+			foreach (var file in files)
+			{
+			    await NavigationHelpers.OpenPath(file, _pageContext.ShellPage!);
+			}
+			
+			foreach (var folder in folders)
+			{
+			    await NavigationHelpers.OpenPath(folder, _pageContext.ShellPage!);
```

Comment:
Isn't this supposed to be `OpenPathInNewTab`?

Reply:
You're right